AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -33,6 +33,27 @@ typedef CALayer *(^ASDisplayNodeLayerBlock)();
 typedef void (^ASDisplayNodeDidLoadBlock)(ASDisplayNode *node);
 
 /**
+ Interface state is available on ASDisplayNode and ASViewController, and
+ allows checking whether a node is in an interface situation where it is prudent to trigger certain
+ actions: measurement, data fetching, display, and visibility (the latter for animations or other onscreen-only effects).
+ */
+
+typedef NS_OPTIONS(NSUInteger, ASInterfaceState)
+{
+  /** The element is not predicted to be onscreen soon and preloading should not be performed */
+  ASInterfaceStateNone          = 1 << 0,
```

Comment:
Could we have this be raw value 0? That way when it's imported into swift, both `[]` and `.None` will correspond to the same value.